frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -241,8 +242,7 @@ function ReaderPaging:addToMainMenu(menu_items)
         text = _("Invert page turn taps and swipes"),
         checked_func = function() return self.inverse_reading_order end,
         callback = function()
-            self.inverse_reading_order = not self.inverse_reading_order
-            self:setupTapTouchZones()
+            ReaderGesture:onToggleReadingOrder()
```

Comment:
should i call this directly like now or through the event system?

Reply:
These days I definitely prefer the Event system in cases like this. I think it makes the code easier to use and refactor.